source/_integrations/dnsip.markdown

```diff
@@ -25,5 +25,7 @@ The **DNS IP** {% term integration %} will expose an IP address, fetched via DNS
 
 {% configuration_basic %}
 Resolver:
-  description: "You may override the default nameservers that are being used by setting any nameserver you like."
+  description: "You may override the default IPV4 and IPV6 nameservers that are being used by setting any nameserver IP address you like, for example `1.1.1.1` (IPV4) or `2606:4700:4700::1111` (IPV6)."
+Port:
+  description: "You may override the default DNS port of `53`. This may be useful to bypass DNS filtering or redirection."
```

Comment:
Would it be worth mentioning that this applies to IPv4 addresses only?

Reply:
It doesn't. It's for both ipv4 and ipv6. 
There is a resolver and a port for each but it's not duplicated here as it's only a descriptive information about the fields.

Reply:
Thank you for clarifying!